baybe/constraints/base.py

```diff
@@ -279,10 +280,17 @@ def to_botorch(
 
         return (
             torch.tensor(param_indices),
-            torch.tensor(self.coefficients, dtype=DTypeFloatTorch),
-            np.asarray(self.rhs, dtype=DTypeFloatNumpy).item(),
+            torch.tensor(
+                [self._multiplier * c for c in self.coefficients], dtype=DTypeFloatTorch
+            ),
+            np.asarray(self._multiplier * self.rhs, dtype=DTypeFloatNumpy).item(),
         )
 
+    @property
```

Comment:
I'd move that to a more prominent place of the class definition as I would assume to find it rather close to the top instead of at the end.

Reply:
moved it up